### PR TITLE
Fixes #955: Added canPushLatestTag input field that allows the user t…

### LINKED ISSUE #955: Include a boolean flag to push :latest tag in image-manual.yml
`image-manual.yml` provides a convenient way of manually re-spinning images. It does however, not allow for pushing the :latest tag. Add a boolean flag in the Run Workflow form to allow for pushing the :latest tag along with the specified tag.

### PR DESCRIPTION
**Pull request by ganeshmurthy**
…o push

the :latest tag

### CODE REVIEW

**Review by jiridanek — APPROVED** [review]

**Comment by jiridanek** [comment]
About CI, I'm seeing there https://github.com/skupperproject/skupper-router/actions/runs/4167633719/jobs/7213457157#step:30:8856

```
E       AssertionError: b'abcd' != b'' : ncat command returned invalid data, expected b'abcd' but got b''
```

is this something that recently appeared again? I remember something like this was already fixed in https://github.com/skupperproject/skupper-router/issues/391

File: .github/workflows/image-manual.yml [hunk 1]
```diff
@@ -29,6 +29,11 @@ on:
         description: 'Docker tag to use for the skupper-router image'
         required: true
         default: 'custom'
+      canPushLatestTag:
+        description: 'Push :latest tag?'
+        required: true
+        type: boolean # default is false for type boolean
```
[the remark this answers is not in the document]
Comment (ganeshmurthy) [reply .github/workflows/image-manual.yml:35]: Done ! Thanks

File: RELEASE.adoc [hunk 2]
```diff
@@ -46,3 +46,14 @@ under the License
   will be unable to assign any more issues to the closed milestone.
 * Create the next milestone if necessary.
 * There is no need to advance any versions on the main branch.
+
+# Re-spinning an already released skupper-router version to include CVE changes from other component/base images
+
+* Sometimes there is a need to re-release an already released version of skupper-router, for example to
+  include any CVE fixes included in one of the component/base images. In this case, we use the same version number as
+  the earlier released version since there are no changes to the skupper-router code base.
+* Before re-spinning an image, push a test tag and create a test image using the "Run Workflow" button the "Manual skupper-router image build"
```
[the remark this answers is not in the document]
Comment (ganeshmurthy) [reply RELEASE.adoc:55]: Done ! Thanks.

File: .github/workflows/image-manual.yml [hunk 3]
```diff
@@ -46,12 +51,19 @@ jobs:
...
           # build, run tests and if all worked publish a custom image
           export PROJECT_TAG="${DOCKER_TAG}"
           # Use {GITHUB_SHA::8} for shorter SHA
           export VERSION="${GITHUB_SHA}"
+          if [ "$CAN_PUSH_LATEST" == "true" ]; then
+            echo 'image-manual.yml setting PUSH_LATEST=true, will push :latest tag'
+            export PUSH_LATEST=true
+          else
+            echo 'image-manual.yml not setting PUSH_LATEST, will not push :latest tag'
+          fi
```
[the remark this answers is not in the document]
Comment (ganeshmurthy) [reply .github/workflows/image-manual.yml:66]: I am not changing anything in here at the moment.

**Comment by ganeshmurthy** [comment]
> About CI, I'm seeing there https://github.com/skupperproject/skupper-router/actions/runs/4167633719/jobs/7213457157#step:30:8856
> 
> ```
> E       AssertionError: b'abcd' != b'' : ncat command returned invalid data, expected b'abcd' but got b''
> ```
> 
> is this something that recently appeared again? I remember something like this was already fixed in #391

It does indeed look the same. Please create a github issue and I will take a look. Thanks.

File: .github/workflows/image-manual.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jiridanek) [reply .github/workflows/image-manual.yml:67]: sure

**Comment by jiridanek** [comment]
Reported https://github.com/skupperproject/skupper-router/issues/957

**Comment by ganeshmurthy** [comment]
> Reported #957

Thank you very much !!

**Pull request merged**